### PR TITLE
Update site to add completely revamped design, mobile support, and better images for sponsors.

### PR DESCRIPTION
**Pull request by vidhanio**
The site is up on [GitHub Pages](https://vidhanio.github.io/4308-site/) for testing. Please let me know if anything needs to be updated! :smile:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,121 +1,164 @@
...
 
-    <hr class="zbm">
-    <footer style="text-align: center;">
-        <a href="https://www.facebook.com/frc4308" style="color: #3b5998"><i class="bx bxl-facebook-square bx-lg"></i></a>
-        <a href="https://twitter.com/frcteam4308" style="color: #1da1f2"><i class="bx bxl-twitter bx-lg"></i></a>
-        <a href="https://www.youtube.com/channel/UCFOy9P3ZDVzclLQF3w-mtsw" style="color: #ff0000"><i class="bx bxl-youtube bx-lg"></i></a>
-        <a href="https://www.instagram.com/frc4308/" style="color: #c13584"><i class="bx bxl-instagram bx-lg"></i></a>
-        <a href="mailto:the.woodlands.robotics@gmail.com"><i class="bx bx-mail-send bx-lg"></i></a>
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```
Comment (wozeparrot) [inline index.html]: Can this entire line be removed as it looks unprofessional.

File: past-seasons.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+                class="event-image"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```
Comment (wozeparrot) [inline past-seasons.html]: Can this entire line be removed as it looks unprofessional.

File: sponsor-us.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+            <i class="fas fa-check list-row"></i>
+            <i class="fas fa-check list-row"></i>
+            <i class="rounded-br-lg fas fa-check list-row"></i>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```
Comment (wozeparrot) [inline sponsor-us.html]: Can this entire line be removed as it looks unprofessional.

File: sponsors.html [hunk 4]
```diff
@@ -1,97 +1,145 @@
...
 
-    <hr class="zbm">
-    <footer style="text-align: center;">
-        <a href="https://www.facebook.com/frc4308" style="color: #3b5998"><i class="bx bxl-facebook-square bx-lg"></i></a>
-        <a href="https://twitter.com/frcteam4308" style="color: #1da1f2"><i class="bx bxl-twitter bx-lg"></i></a>
-        <a href="https://www.youtube.com/channel/UCFOy9P3ZDVzclLQF3w-mtsw" style="color: #ff0000"><i class="bx bxl-youtube bx-lg"></i></a>
-        <a href="https://www.instagram.com/frc4308/" style="color: #c13584"><i class="bx bxl-instagram bx-lg"></i></a>
-        <a href="mailto:the.woodlands.robotics@gmail.com"><i class="bx bx-mail-send bx-lg"></i></a>
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```
Comment (wozeparrot) [inline sponsors.html]: Can this entire line be removed as it looks unprofessional.

**Review by wozeparrot — CHANGES_REQUESTED** [review]
Just some minor changes needed to make it look more professional.

File: sponsors.html (hunk 4 above)
Comment (vidhanio) [reply sponsors.html]: Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.

File: sponsor-us.html (hunk 3 above)
Comment (vidhanio) [reply sponsor-us.html]: Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.

File: past-seasons.html (hunk 2 above)
Comment (vidhanio) [reply past-seasons.html]: Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.

File: index.html (hunk 1 above)
Comment (vidhanio) [reply index.html]: Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.

**Review by wozeparrot — APPROVED** [review]
LGTM

**Pull request merged**